app/javascript/controllers/autocompleter_controller.js

```diff
@@ -212,6 +212,7 @@ export default class extends Controller {
   // to refresh the primer (as with location_containing a changed lat/lng).
   //
   swap({ detail }) {
+    console.log("swap called");
```

Comment:
Note that this Stimulus controller has a method `verbose`.
If you switch your `console.log("swap called")` to `this.verbose("swap called"), and then uncomment the actual method body of `verbose`, it will log a whole trace of what it's executing (as defined by wherever Jason and I left lines that say 
```js 
this.verbose("Say something");
```